src/plugins/irc-events/ctcp.js

```diff
@@ -22,7 +25,8 @@ module.exports = function(irc, network) {
 		chan.pushMessage(client, msg);
 	});
 
-	irc.on("ctcp request", (data) => {
+	// Limit requests to a rate of one per second max
+	irc.on("ctcp request", _.throttle((data) => {
```

Comment:
Mmmmh, cba honestly.

Did you see my comment in the thread:

> @xPaw, added a limit of 1 request per second. I wanted to do 5 max per 5s, but lodash comes with a "1 per N ms" throttling mechanism and I didn't want to write my own, so that will do.

I don't really want to implement that myself, however simple it is. I can reduce the `1000`ms value though :man_shrugging: